### ISSUE TITLE
Maps defaulting to PEI

### ISSUE THREAD

**Issue by gabiro16** [SmartlyDressedGames/Unturned-3.x-Community#4226]
my steamcmd server maps are all defaulting to PEI for some reason after the last patch, pls help

**Comment by Sgtlog123** [comment]
did you have workshop install? or did you specify the map in command.dat?

**Comment by SDGNelson** [comment]
Where is your map installed? Are you on the latest .1 patch?

**Comment by gabiro16** [comment]
> Where is your map installed? Are you on the latest .1 patch?

Its installed both in server workshop maps and normal maps

**Comment by gabiro16** [comment]
Im almost sure that it is already on .1 patch, how do I confirm it?

> did you have workshop install? or did you specify the map in command.dat?

Yes, I have the map installed on workshop and its specified in command.dat

**Comment by gabiro16** [comment]
> Where is your map installed? Are you on the latest .1 patch?

Yes, my Unturned is already on the latest .1 patch

**Comment by gabiro16** [comment]
> Can you attach your server's log file please?

Sure.
[Server_PEIModifiedv2.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/13479890/Server_PEIModifiedv2.log)

Weird think is that at first, it appears that PEI Modified was found and set, but in the end of the cmd, for some reason, it fails to find the map. Perhaps that is has something to do with this command line

k_ESteamNetworkingAvailability_Waiting "Waiting for Steam login"
[2023-11-26 23:40:43] Steam servers ready!
[2023-11-26 23:40:43] Workshop install folder: C:\Users\GabiroPC\Desktop\steamcmd\steamapps\common\U3DS/Servers/PEIModifiedv2/Workshop/Steam
[2023-11-26 23:40:43] 0 workshop item(s) requested

**Comment by SDGNelson** [comment]
It looks like it isn't finding any extra maps the vanilla ones in `C:\Users\GabiroPC\Desktop\steamcmd\steamapps\common\U3DS\Maps`

Judging by `0 workshop item(s) requested` your `WorkshopDownloadConfig.json` file is empty. Maybe it got reset at some point?

**Comment by gabiro16** [comment]
> It looks like it isn't finding any extra maps the vanilla ones in `C:\Users\GabiroPC\Desktop\steamcmd\steamapps\common\U3DS\Maps`
> 
> Judging by `0 workshop item(s) requested` your `WorkshopDownloadConfig.json` file is empty. Maybe it got reset at some point?

yeaaah, I figured how to fix it last night, by seeing another reddit person with the same issue I analyzed the error code line and concluded that it WorkshopDownloadConfig.json was empty and I had to put the map ID number into the brackets again. Btw, Thank you Nelson!

**Comment by gabiro16** [comment]
> It looks like it isn't finding any extra maps the vanilla ones in `C:\Users\GabiroPC\Desktop\steamcmd\steamapps\common\U3DS\Maps`
> 
> Judging by `0 workshop item(s) requested` your `WorkshopDownloadConfig.json` file is empty. Maybe it got reset at some point?

yeaaah, I figured how to fix it last night, by seeing another reddit person with the same issue I analyzed the error code line and concluded that it WorkshopDownloadConfig.json was empty and I had to put the map ID number into the brackets again. 

Apparently the last update emptied/reseted some server's WorkshopDownloadConfig.json, since a few other people in reddit are reporting the same issue as I. Anyways, thank you Nelson!
